CarAccidentScenario/InsuranceClaimException.java

```diff
@@ -0,0 +1,8 @@
+package CarAccidentScenario;
+
+public class InsuranceClaimException extends Exception {
```

Comment:
This name is a bit confusing as I am unable to understand what the error is, is it error in filing claim or is it error in getting the claim or something else.
Please rename it to something more specific.